hack/tilt/setup.sh

```diff
@@ -0,0 +1,508 @@
+#!/bin/bash
+
+TILT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+SCRIPT_DIR=$(dirname $TILT_DIR)
+BIN_DIR=$(dirname $SCRIPT_DIR)/temp/tilt/bin 
+
+CLUSTER_NAME="kpng-proxy"
+KINDEST_NODE_IMAGE="docker.io/kindest/node"
+KIND_VERSION="v0.14.0"
+K8S_VERSION="v1.25.0"
+OS=$(uname| tr '[:upper:]' '[:lower:]')
+
+CLUSTER_CIDR_V4="10.1.0.0/16"
+SERVICE_CLUSTER_IP_RANGE_V4="10.2.0.0/16"
+CLUSTER_CIDR_V6="fd6d:706e:6701::/56"
+SERVICE_CLUSTER_IP_RANGE_V6="fd6d:706e:6702::/112"
+KPNG_SERVER_ADDRESS="unix:///k8s/proxy.sock"
+KPNG_DEBUG_LEVEL=4
+KIND="kindest/node:v1.22.13@sha256:4904eda4d6e64b402169797805b8ec01f50133960ad6c19af45173a27eadf959"
```

Comment:
same for these vars, should be on tilt.env

Reply:
+1

Reply:
Currently, only the below variable is in `setup.sh`. others were moved to `common.sh`.
```console
KIND="kindest/node:v1.22.13@sha256:4904eda4d6e64b402169797805b8ec01f50133960ad6c19af45173a27eadf959"

```